tests/e2e/test_actions.py

```diff
@@ -2,14 +2,15 @@
 Module with tests for operators
 """
 import logging
+import os
 import subprocess
 
 import pytest
 
 from . import utils
 
 LOGGER = logging.getLogger(__name__)
-DEFAULT_TIMEOUT = 15
+DEFAULT_TIMEOUT = int(os.environ.get("EDA_E2E_DEFAULT_TIMEOUT", "15"))
```

Comment:
As a preparation for https://issues.redhat.com/browse/AAP-9026 which is a previous step of https://issues.redhat.com/browse/AAP-9047 I would implement a config.py and a settings instance for the e2e tests and manage the default timeout in only place.

Reply:
Maybe the refactor of E2E tests should come in a different PR.